rosette/rbl/rosette/space.rbl

```diff
@@ -86,7 +86,18 @@
     ;;; TODO: We currently do not support prologue style unification (bidirectional) pattern matching
     ;;; Potential implementation: (if (type? ptrn candidate) (seq (update) ((proc [ptrn] ptrn) candidate)) (seq (update) #niv)) )
     (pure (unify? ptrn candidate) #t)
-    (pure (match? ptrn candidate) (if (type? ptrn Symbol) #t (type? ptrn candidate)))
+    (pure (match? ptrn candidate)
+      (if (type? ptrn Symbol)
+        #t
+        (if (type? ptrn candidate)
+          (cond
+            ((type? ptrn String)
+              (= ptrn candidate))
+            ((type? ptrn Tuple)
+              (proc [[hp & ptrn] [cp & candidate]] (and (= hp cp) (match? ptrn candidate))) ptrn candidate)
```

Comment:
You're probably fixing this as I type this, but you're not running the procedure anymore; this clause will just return `candidate`.

Reply:
Yep, fixed in my most recent upload.